board/safety/safety_mazda.h

```diff
@@ -12,12 +12,11 @@
 #define MAZDA_AUX  1
 #define MAZDA_CAM  2
 
-#define MAZDA_MAX_STEER 2048U
+#define MAZDA_MAX_STEER 800U
```

Comment:
@Jafaral Do you know why this was set to 2048 if the limit in openpilot is only 800? I read here https://github.com/commaai/openpilot/pull/20864 you said newer models have more torque, but still don't respond above certain values, so think this is okay to correctly set?

Reply:
I did see values north of 1500 in newer  models. @adeebshihadeh  suggested we keep the max at 800 in OP. Matching that in panda is probably fine.

Reply:
Sounds good! If we ever want to verify these higher values are safe in openpilot we can always raise the limits and fix the `test_steer_safety_check` check (double 1500 would put it above the max signal value, wrapping back around to negative values)

Reply:
The 2048 is the theoretical maximum. Sending high torques values is OK, the car trims these to whatever is allowed in the EPS without causing faults/errors.

Reply:
I think the range is actually -2048 to 2047 here, 2048 would be -2048

Reply:
you are correct about the range. I  just used a proper power of two so that I can use the constant as-is in the code which you updated also in the PR. All looking good.